x-pack/plugins/security_solution/public/timelines/components/timeline/search_super_select/index.tsx

```diff
@@ -106,14 +106,16 @@ const SearchTimelineSuperSelectComponent: React.FC<SearchTimelineSuperSelectProp
               description: t.description,
               favorite: t.favorite,
               label: t.title,
-              id: t.savedObjectId,
+              id: timelineType === TimelineType.default ? t.savedObjectId : t.templateTimelineId,
```

Comment:
If a new value (e.g. `superTimeline`) is added to the `TimelineType` enum, this expression as it's written would return `t.templateTimelineId` instead of `t.savedObjectId` when `TimelineType` is `superTimeline`, which is probably not the intended behavior.

Consider this safer alternative:

```
id: timelineType === TimelineType.template ? t.templateTimelineId : t.savedObjectId,
```